### DISCUSSION TITLE
Why doesn't the theme get applied to the pop-up of the  Top Bar, or the system menu?

### DISCUSSION THREAD

**Discussion by HubKing** [GradienceTeam/Gradience#703] [Q&A] [answered]
The theme does not seem to be applied to the pop-up elements of the Top Bar or the system menu. Is this a limitation of Gnome that cannot be circumvented?

![image](https://user-images.githubusercontent.com/10900489/212984804-93b3decf-cb88-4e39-b53f-465353f6ae06.png)

**Comment by daudix** [comment, marked as answer]
This is work in progress, see #679
